tests/test_bot.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+import asyncio
+import random
+import sqlalchemy as sa
+
+from app.models import Bot
+
+'''Tests must be run together'''
+
+
+@pytest.mark.order(1)
+@pytest.mark.asyncio
+async def test_create_delete(client, container):
+    create_one = await client.post("/bot", json={"secret_id": "sercret123",
+                        "client_id": "client123", "nickname": "Cooper123"})
+    assert create_one.status_code == 200
+
+    query = await container.bot.query()
```

Comment:
we don't check here if created_one is inside db by doing query.get(id) and assert for example client_id in json request and db